.circleci/config.yml

```diff
@@ -916,6 +923,24 @@ jobs:
       - turf-report-failure:
           turf-revision: << parameters.turf-revision >>
 
+  trigger-internal-ci-sync:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - run:
+          name: Notify maps internal CI about $CIRCLE_BRANCH branch update
+          command: |
+            curl -X POST "https://circleci.com/api/v2/project/github/mapbox/mapbox-maps-ios-private/pipeline" \
+                -H 'Content-Type: application/json' \
+                -H "Circle-Token: $CIRCLECI_API_TOKEN_MAPBOX_MAPS_IOS_INTERNAL" \
+                -d $'{
+                        "branch": "main",
+                        "parameters": {
+                          "mapbox_sync_branch": "$CIRCLE_BRANCH"
```

Comment:
I have some light concerns about $CIRCLE_BRANCH. Most of the time we would trigger build system with the `CIRCLE_BRANCH=main`. It means that potentially, there are might be parallels builds in the `ios-private` repo. 
Could it be an issue?